src/tools/rbd/action/ObjectMapCheck.cc

```diff
@@ -0,0 +1,74 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Can this just be combined with the current "ObjectMap.cc" file to keep all object map operations together?